### ISSUE TITLE
[Accessibility]: MeterChart - The 'id' "modal-title, modal-description" specified for the ARIA property 'aria-labelledby, aria-describedby' value is not valid 

### ISSUE THREAD

**Issue by trisgerra** [carbon-design-system/carbon-charts#1459]
### Contact Details

andrea.digiacomo@live.it

### Environment

Browser

### What happened? What did you expect to happen instead?

When i run IBM Equal Access Accessibility Checker in a page with a MeterChart, I see the violation in the title repeated for the number of charts in the page.
![Screenshot 2022-10-28 at 11 37 14](https://user-images.githubusercontent.com/3984060/198557335-c9fb0444-d34a-4b6d-809e-dcefb6b35d81.png)

### What WCAG 2.1 checkpoint does the issue violate?

https://www.w3.org/WAI/WCAG21/Understanding/name-role-value

### Version

@carbon/charts": "^1.6.1

### Data & options used

```js
{
  "data": [
    "{group: \"Group1\", value: 10}",
    "{group: \"Group2\", value: 28}",
    "{group: \"Group3\", value: 11}",
    "{group: \"Group4\", value: 0}"
  ],
  "options": {
    meter: {
                showLabels: false,
                proportional: {
                    total: total || 1,
                    unit: 'unit'
                }
            },
            height: '400px,
            color: {
                scale: {
                    Group1: '#ffb3b8',
                    Group2: '#f1c21b',
                    Group3: '#ff832b',
                    Group4: '#a2191f'
                }
            },
            legend: {
                alignment: Alignments.RIGHT,
                clickable: true
            },
            toolbar: {
                enabled: false
            },
            tooltip: {
                enabled: true
            },
            theme: ChartTheme.G90
        };
  }
}
```

**Comment by theiliad** [comment]
Followed up with Andrea, and it seems like the issue was that the `@carbon/styles/css/styles.css` import was missing.

**Comment by trisgerra** [comment]
I confirm that the issue has been resolved by importing the style above.
It needs to be imported also if the toolbar is disabled in the chart options.

If you have the same accessibility issue related to MeterChart, try to enable the toolbar and check if the modal is rendered correctly. 
In the case the buttons are without css and the modal is transparent the problem is probably related to the missing import.
